src/bg/user-script-detect.js

```diff
@@ -0,0 +1,103 @@
+/*
+Add listeners to detecting user scripts and opening the installation
```

Comment:
This form:

    /* Add listeners to detect user scripts and open the installation dialog. */

Is better grammar and fits on one line.